internal/build/env.go

```diff
@@ -45,11 +46,12 @@ type Environment struct {
 	Buildnum                  string
 	IsPullRequest             bool
 	IsCronJob                 bool
+	StaticLink                bool
```

Comment:
This is the wrong place to put this config option. The `Environment` contains information about the CI system and source tree. It does not configure the build itself.

You should add the command-line flag in `ci.go` instead.

Reply:
Okay, I'll delete it